browser-tests/functional-tests/accessibility.spec.js

```diff
@@ -0,0 +1,32 @@
+const { test, expect } = require("@playwright/test");
+const { iteratePagesAndContexts } = require("../data/pagesAndContexts.js");
+const AxeBuilder = require("@axe-core/playwright");
+
+test.describe.parallel("Accessibility tests", () => {
+  test.setTimeout(120000);
+  iteratePagesAndContexts(
+    (pageName, context, language, url) => {
+      test(`Accessibility check for ${pageName}, context ${context} and language ${language}`, async ({ page }) => {
+        await page.goto(url);
+        await assertNoAccessibilityViolations(page);
+      });
+    }
+  );
+
+  (["radio", "checkbox"]).forEach((errorState) => {
+    ["en", "cy"].forEach((language) => {
+      test(`Accessibility check for ${errorState} error state in ${language}`, async ({ page }) => {
+        await page.goto(`http://localhost:4601/dev/template/confirm-your-details/${language}?errorState=${errorState}`);
```

Comment:
Probably best to get the domain for this from config or at least define it in one place somewhere?

Reply:
Also what does setting the error state actually do on this specific template 🤔

Reply:
I thought that myself and commented the same thing on the Snapshot tests PR. This was the response: [thread](https://github.com/govuk-one-login/ipv-core-front/pull/1574#discussion_r1758825390), so I've elected to keep it the same here.

On your second point, I wrote a bit on it in the [README for pages on creating pages](https://github.com/govuk-one-login/ipv-core-front/blob/main/src/views/README.md#:~:text=pageErrorState,/%20radiobox). Our error handling has redundancy - we have _two_ objects which lead to error states, but this is the only one where can specify whether it is the radio or checkbox which is failing.

Reply:
Can't we pull it from the env vars defined in the docker compose file?